content/working_with/console/widgets/highAvailability.md

```diff
@@ -1,13 +1,33 @@
 ---
 layout: bt_wiki
-title: High Availability
+title: Cluster Status
 category: Cloudify Console
 draft: false
 ---
 
-Displays the Manager’s High-Availability status. If a cluster architecture is configured on the manager, this widget will show the cluster-connected nodes. There is no click-through actions available from this widget, as all Cluster management actions should be performed from the Cloudify CLI / REST API. 
+Displays status of the Cloudify cluster divided into 3 cluster services: 
+
+* Manager, 
+* Database,
+* Message Broker. 
+
+![cluster-status-widget]( /images/ui/widgets/cluster-status.png )      
```

Comment:
Leaving the comment here, but the question is about the cluster-status png - I dont think there is value in showing 127.0.0.1 as the internal IP address. also, if not too complex, can you edit the public IP to be different than the private one?

Reply:
Fixed using mock.